errors.go

```diff
@@ -93,11 +93,20 @@ type ValidationError struct {
 }
 
 func (e ValidationError) Error() string {
+	var location string
+
 	if e.Field == "" {
-		return fmt.Sprintf("line %d: %s", e.Line, e.Err)
+		location = fmt.Sprintf("line %d", e.Line)
+	} else {
+		location = fmt.Sprintf("line %d, field %s", e.Line, e.Field)
+	}
+
+	if e.Context != nil {
+		return fmt.Sprintf("%s: %s: %v", location, e.Err, e.Context)
+	} else {
+		return fmt.Sprintf("%s: %s", location, e.Err)
```

Comment:
Is it preferred to not wrap the last `return` statement in an `else` block so it is clear what the last resort is.